components/FormField.tsx

```diff
@@ -1,9 +1,39 @@
-export default function FormField({ name, label, type = "text" }) {
-  return (
-    <div>
-      <label htmlFor={name}>{label}</label>
+interface FormProps {
+  name: string;
+  label: string;
+  textarea?: boolean;
+  type?: string;
+}
+
+export default function FormField({
+  name,
+  label,
+  textarea,
+  type = "text",
+}: FormProps) {
+  return textarea ? (
+    <div className="w-full my-2 leading-4">
```

Comment:
Maybe it'd make sense to have two components? One that's a TextField and one that's a TextArea?